shell.nix

```diff
@@ -16,7 +16,10 @@ let
     };
     hooks = {
       stylish-haskell.enable = true;
-      nixpkgs-fmt.enable = true;
+      nixpkgs-fmt = {
+        enable = true;
+        excludes = [ ".*nix/stack.materialized/.*" ".*nix/sources.nix$" ];
```

Comment:
Worth a comment explaining why we need this as well as the `.ignore` and that they need to be in sync.